api/transport/handler.go

```diff
@@ -119,62 +115,50 @@ type StreamHandler interface {
 
 // DispatchUnaryHandler calls the handler h, recovering panics and timeout errors,
 // converting them to yarpc errors. All other errors are passed trough.
+// Deprecated: Use InvokeUnaryHandler instead.
```

Comment:
I think you need to add an empty line above this for staticcheck to treat is as a deprecated annotation.

```
//
// Deprecated: Use InvokeUnaryHandler instead.
```